proto/spacemesh/types.proto

```diff
@@ -0,0 +1,162 @@
+syntax = "proto3";
+package spacemesh;
+
+message SimpleInt {
+    uint64 value = 1;
+}
+
+message SimpleString {
+    string value = 1;
+}
+
+// a non-negative coin amount
+message Amount {
+    uint64 value = 1;
+}
+
+message AccountId {
+    bytes address = 1;
+}
+
+message Account {
+    AccountId address = 1;  // account public address
+    bytes public_key = 2;   // nice to include the account full public key
```

Comment:
I think I'd leave this out, it bloats the data and the API consumer should be agnostic to this

Reply:
Do you mean the public key field? It is for the explorer. We need to display these in it.

Reply:
Why should the explorer display public key? Why should it display keys at all? It only needs to display addresses. I've never seen an explorer that displays keys :)

Reply:
What I'm trying to say is that I think the Account should be agnostic to how we implement keypairs. Maybe we decide to change the keys in the future (quantum resistance? multiple types of keys?).